webpack.config.js

```diff
@@ -97,6 +97,13 @@ module.exports = {
 					10
 				) || 1
 			),
+			// Inject the `COMPONENT_SYSTEM_PHASE` global, used for controlling Component System roll-out.
+			'process.env.COMPONENT_SYSTEM_PHASE': JSON.stringify(
+				parseInt(
+					process.env.npm_package_config_COMPONENT_SYSTEM_PHASE,
+					10
+				) || 1
```

Comment:
Noting that `COMPONENT_SYSTEM_PHASE` is enabled at the moment.

When I disable it I see the following error when trying to insert the Paragraph block:

![Uploading Screen Shot 2020-12-09 at 21.41.48.png…]()

Reply:
I can confirm it's resolved now 👍🏻